src/main/index.ts

```diff
@@ -148,7 +161,12 @@ if (!isFirstInstance) {
   app.on('session-created', (session) => {
     if (!isDevelopment) {
       log.info('Cleaning local storage on session creation');
+
+      reservedStateValues = reservedStates.map((state) => localStorage.getItem(state));
```

Comment:
I don't think `localStorage` exists in the main process. You should execute code in the renderer process from the main process. Example: https://stackoverflow.com/a/57765134.